src/config/config.ts

```diff
@@ -12,7 +10,6 @@ export interface StoreConfig {
     currency: StoreCurrency;
     displayDateFormat: string;
     inputDateFormat: string;
-    formFields: FormFields;
```

Comment:
Removing `formFields` from the interface would be a breaking change though? It's okay to remove the field from the API because the config endpoint is a private endpoint. But from SDK perspective, `Config` interface is not a private interface so we can't just remove it. 

In order to maintains backwards compatibility, perhaps we can use the data from the new form-field endpoint and just map it to the existing `Config` interface?

Reply:
That's a good point. However, I doubt clients would be consuming these fields directly rather than using the selector which takes care of populating states and provinces. Might be a bit late, but might be worth tagging the data returned in the store config as "beta" or "internal only"? thoughts?

Reply:
Maybe, but we can only guess. It's like making an API change hoping nobody's relying on the change 😄. I supposed it's a different story if we can conditionally exclude the change from people who depend on it. Anyway, I think it should be fairly simple change and I can see you've already started working on it.

In regards to adding `beta` and `internal only` to `Config`, I don't think we should? Our Open Checkout depends on it. And it does surface data that are necessary to build a checkout (otherwise we won't be using it). We can, however, add a `deprecated` annotation to the `formFields` field and suggest people to use `getShippingAddressFields`, `getBillingAddressFields` etc... instead.

Reply:
I have added `@deprecated` tag for now, since it cannot really be made "internal only" if it's been part of the public API.